lib/CompletionExtension.php

```diff
@@ -81,9 +83,23 @@ private function registerCompletion(ContainerBuilder $container): void
             return new TypedCompletorRegistry($mapped);
         });
 
-        $container->register(self::SERVICE_FORMATTER, function (Container $container) {
+        $container->register(self::SERVICE_SHORT_DESC_FORMATTER, function (Container $container) {
             $formatters = [];
-            foreach (array_keys($container->getServiceIdsForTag(self::TAG_FORMATTER)) as $serviceId) {
+            foreach (array_keys($container->getServiceIdsForTag(self::TAG_SHORT_DESC_FORMATTER)) as $serviceId) {
+                $taggedFormatters = $container->get($serviceId);
+                $taggedFormatters = is_array($taggedFormatters) ? $taggedFormatters : [ $taggedFormatters ];
+
+                foreach ($taggedFormatters as $taggedFormatter) {
+                    $formatters[] = $taggedFormatter;
+                }
+            }
+
+            return new ObjectFormatter($formatters);
+        });
+
+        $container->register(self::SERVICE_SNIPPET_FORMATTER, function (Container $container) {
```

Comment:
FYI I'm moving passively towards using class FQNs for service names, but that's not applicable here.

Reply:
I totally agree with that :)